src/ert/shared/models/base_run_model.py

```diff
@@ -99,7 +99,8 @@ def __init__(
         self.reset()
 
         # experiment-server
-        self._state_machine = ExperimentStateMachine()
+        self._state_machine = com_protocol.ExperimentStateMachine()
+        self._iter_map: Dict[int, str] = {}
```

Comment:
Should we state that this is a mapping from iteration numbers to ensemble id's (unless we can come up with a more descriptive name)

Reply:
Will add comment